tests/test_repo_legit.py

```diff
@@ -0,0 +1,126 @@
+from os.path import expanduser, isdir, exists, join, abspath, dirname
```

Comment:
please don't import stdlib methods with generic names like `join` directly, do something like 
```suggestion
from os import path
```
if you want to shorten the calls a little.